tmt/steps/discover/__init__.py

```diff
@@ -269,6 +279,9 @@ def extract_distgit_source(
             stdout, _ = self.run(
                 ["git", "config", "--get-regexp", '^remote\\..*.url'],
                 cwd=distgit_dir)
+            if stdout is None:
+                raise tmt.utils.GeneralError("Missing remote origin url.")
+
```

Comment:
why did you add this?

Reply:
See @thrix's comment above. He asked me to add this error result.

Reply:
+1